src/library/scala/collection/ArrayOps.scala

```diff
@@ -558,6 +558,34 @@ final class ArrayOps[A](val xs: Array[A]) extends AnyVal {
   }
 
 
+  /** Applies a binary operator to a start value and all elements of this array,
+    * going right to left, keeping list of intermediate values.
```

Comment:
This description is not very clear. What do you think of reusing the existing description? https://github.com/scala/scala/blob/v2.12.5/src/library/scala/collection/GenTraversableLike.scala#L126-L138

Reply:
Good idea, it was a copy paste from foldLeft adding a sentence about intermediate values. I think simple example on existing description explains better than x_1, x_2